cmd/contour/serve.go

```diff
@@ -137,7 +137,9 @@ func registerServe(app *kingpin.Application) (*kingpin.CmdClause, *serveContext)
 	serve.Flag("disable-leader-election", "Disable leader election mechanism.").BoolVar(&ctx.LeaderElection.Disable)
 
 	serve.Flag("envoy-http-access-log", "Envoy HTTP access log.").PlaceHolder("/path/to/file").StringVar(&ctx.httpAccessLog)
+	serve.Flag("envoy-http-buffer-max-request-bytes", "Envoy HTTP buffer max request bytes.").PlaceHolder("10485760").Uint32Var(&ctx.httpBufferMaxRequestBytes)
```

Comment:
i dont think we want to add command line flags for this, rather for now just add configuration options to the configuration CRD and config file schema (https://github.com/projectcontour/contour/blob/main/pkg/config/parameters.go)

is there a reason you want to add a separate limit for http vs. https?

Reply:
> for now just add configuration options to the configuration CRD and config file schema
ok, I'll try to do that. 

> is there a reason you want to add a separate limit for http vs. https?
no, there wasn't any specific reason behind it; it was just a proposal.  If you believe that merging them into a single parameter would be more effective, then let's proceed with that approach.

Reply:
fixed